vm/actor/src/builtin/power/types.rs

```diff
@@ -18,17 +18,6 @@ pub const SECTOR_TERMINATION_MANUAL: SectorTermination = 1;
 /// Implicit termination due to unrecovered fault
 pub const SECTOR_TERMINATION_FAULTY: SectorTermination = 3;
 
-// TODO revisit on miner actor completion if duplicate type
-#[derive(Serialize_tuple, Deserialize_tuple)]
-pub struct MinerConstructorParams {
-    pub owner: Address,
-    pub worker: Address,
-    pub control_addrs: Vec<Address>,
-    pub seal_proof_type: RegisteredSealProof,
-    pub peer: BytesDe,
-    pub multiaddrs: Vec<BytesDe>,
```

Comment:
Just to indicate previous comment what the type should look like. (You don't need to use the `BytesDe` type, I just did out of convenience)